### ISSUE TITLE
Support custom user-friendly URLs (slug) for Listed

### ISSUE THREAD

**Issue by nablagodel** [standardnotes/forum#1067]
I want to set custom user-friendly URLs (slug) for my posts in Listed because Listed can't convert non-latin characters.

Example URL: https://listed.to/@nablagodel/17336/.

I suggest to add new per-post option (https://docs.standardnotes.org/listed/options) for example `slug` (https://en.wikipedia.org/wiki/Clean_URL).

```
---
slug: svyazka-instrumentov-dlya-ukroscheniya-informacionnogo-shtorma
---
```

And, as result, we get follow URL: https://listed.to/@nablagodel/17336/svyazka-instrumentov-dlya-ukroscheniya-informacionnogo-shtorma/

**Comment by JaspalSuri** [comment]
Thanks for the feedback! I'll pass this along to the team for review.

**Comment by mobitar** [comment]
You can already do that. Just click your last URL. You can put anything you want after the ID of the post (https://listed.to/@nablagodel/17336/anything-you-want)

**Comment by nablagodel** [comment]
@mobitar, yes, I know it. But I am talking about links in the blog. Its don't include slug.
